charts/sourcegraph/templates/redis/redis-cache.Deployment.yaml

```diff
@@ -53,9 +53,27 @@ spec:
         {{- end }}
         terminationMessagePolicy: FallbackToLogsOnError
         livenessProbe:
-          initialDelaySeconds: 30
+          initialDelaySeconds: 60
+          periodSeconds: 30
+          failureThreshold: 2
+          timeoutSeconds: 5
           tcpSocket:
             port: redis
+        readinessProbe:
+          initialDelaySeconds: 5
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - |
+              #!/bin/bash
+              response=$(
+                redis-cli ping
+              )
+              if [ "$response" != "PONG" ]; then
+                echo "$response"
+                exit 1
+              fi
```

Comment:
@ggilmore made a suggestion on the call yesterday after you left that it might be nice to add this script to the image itself in sg/sg, and we only need to invoke it from here. 
Benefits:
- Portable to all deployment types, without the need to reimplement this every time, and fix it everywhere, should there be more edge-cases we didn't cover
- We can include it in our image tests to make sure the script keeps working

Reply:
Implemented in https://github.com/sourcegraph/sourcegraph/pull/60365